main.c

```diff
@@ -57,6 +57,42 @@ int lenient_strcmp(char *a, char *b) {
 	}
 }
 
+#ifdef ENABLE_READYFD
+static void readyfd(void) {
+	int fd;
+	char *env = NULL, *endptr = NULL;
+
+	env = getenv("READY_FD");
+
+	if (!env || env[0] == '\0') {
```

Comment:
Is it really needed to have an argument for READY_FD? Can't we just enable it if the env variable is defined?

Reply:
I could do that. I don't in this case for two reasons:

1. While I hope this does not happen, the variable could be leaked from a parent process. In such a case the contents are likely invalid or at least not intended to be used by swaylock.

2. If a user configures their service with an older version of swaylock, the supervisor will be twiddling its thumbs while it waits for a notification that will never come. This would lead to some unexpected (although not particularly nasty) behavior. A flag would make such a misconfiguration glaringly obvious (`swaylock: unrecognized option '--readyfd'`).

Reply:
Is this a blocker?

Reply:
No.